src/Standards/PSR12/Docs/Files/ImportStatementStandard.xml

```diff
@@ -0,0 +1,55 @@
+<documentation title="Import Statement">
+    <standard>
+    <![CDATA[
+    Import statements must not begin with a leading backslash.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Import statements are fully qualified and don't begin a leading backslash.">
```

Comment:
The example only shows one import `use` statement, so I think the phrasing should be singular ? (same for the _invalid_ code sample description)

```suggestion
        <code title="Valid: Import statement are fully qualified and don't begin a leading backslash.">
```

Reply:
>  (same for the _invalid_ code sample description)

☝🏻

Reply:
Shoot - just noticed a missing word in the above as well - "with".

```diff
-        <code title="Valid: Import statement are fully qualified and don't begin a leading backslash.">
+        <code title="Valid: Import statement are fully qualified and don't begin with a leading backslash.">
```